### PR TITLE
Affaire UUID

### PR DESCRIPTION
**Pull request by rufener**
add `uuid` field to affaire in order to open unpredictable links for services to get affaires informations.

### CODE REVIEW

File: back/infolica/views/affaire_document.py [hunk 1]
```diff
@@ -108,23 +108,23 @@ def download_affaire_document_view(request):
...
 def download_affaire_document_view(request):
     """
     Download document for SPCH
     """
-    t_request = request.params["t"] if "t" in request.params else None
-    t_config = request.registry.settings["spch_token"]
+
+    affaire_id = request.params["affaire_id"] if "affaire_id" in request.params else None
+    affaire = request.dbsession.query(Affaire).filter(Affaire.id == affaire_id).first()
+    uuid = request.params["uuid"] if "uuid" in request.params else None
```
Comment (kalbermattenm) [inline back/infolica/views/affaire_document.py:119]: Don't you need to add a test to check that `affaire_id` is not none (and raise an error if it is the case)?

**Review by kalbermattenm — APPROVED** [review]
I don't get what you mean to do with this PR...

Why are you adding an UUID which you retrieve through the id?

Is it to hide affaires from being called from somebody you would not know the id AND the UUID of an affair (to add complexity to information retrieval)?

**Comment by rufener** [comment]
> Is it to hide affaires from being called from somebody you would not know the id AND the UUID of an affair (to add comple

Exactly. The idea is that the user cannot predict the url of next affaire. The UUID will also be updated periodically in such a way that an affaire is not accessible with the same url indefinitely.
